spec/miq_ec2_vm/miq_ec2_ebs_instance_spec.rb

```diff
@@ -3,6 +3,9 @@
 require_relative '../spec_helper'
 require_relative '../aws_ssa_commons'
 
+::Module.include(LogDecorator::Logging::ClassMethods)
+include LogDecorator::Logging
```

Comment:
I'm surprised you need this in specs,  but even so, I'd think this belongs in spec_helper.